Source/OCcore/RawDump/rawdump.h

```diff
@@ -0,0 +1,12 @@
+//
+// Created by supragya on 13/4/18.
+//
+
+#ifndef APERTUSOPENCINE_RAWDUMP_H
+#define APERTUSOPENCINE_RAWDUMP_H
+
+namespace OcRawDump{
```

Comment:
Namespaces in OC start usually with namespace OC, then area, like Image or something like that, so it should be OC::Image::RawDump

Reply:
corrected in #38